instrumentation/grpc-1.5/grpc-1.5.gradle

```diff
@@ -57,7 +57,7 @@ dependencies {
   // this instrumentation needs to be able to be able to reference the OpenTelemetry API's gRPC Context
   // that is shaded in the bootstrap class loader (for sending telemetry to the agent),
   // separately from the gRPC Context that is brought by gRPC
-  compileOnly project(path: ':opentelemetry-api-shaded-for-instrumenting', configuration: 'shadow')
+  compileOnly project(path: ':opentelemetry-api-beta-shaded-for-instrumenting', configuration: 'shadow')
```

Comment:
We did not have version here, why should we now?

Reply:
this one i think we should keep, because (at least the way it currently works) we will need one

`opentelemetry-api-X.Y-shaded-for-instrumenting`

module for each

`opentelemetry-api-X.Y`

instrumentation module.